openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py

```diff
@@ -56,8 +57,17 @@ def get_data(series, record_component, i_slice=None, pos_slice=None,
     if i_slice is not None and not isinstance(i_slice, list):
         i_slice = [i_slice]
 
-    chunks = record_component.available_chunks()
+    # perfer chunk-wise reading for ADIOS2
+    if series.backend == 'ADIOS2':
+        chunks = record_component.available_chunks()
+    else:
```

Comment:
From the trace here, it actually looks to me like we are returning only one chunk in HDF5 already:
https://github.com/Hi-PACE/hipace/pull/725#issuecomment-1084950455